modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/ViewContentsSectionDisplayContext.java

```diff
@@ -45,6 +50,9 @@ public ViewContentsSectionDisplayContext(
 			language, objectDefinitionService,
 			objectDefinitionSettingLocalService,
 			objectEntryFolderModelResourcePermission, portal);
+
+		themeDisplay = (ThemeDisplay)httpServletRequest.getAttribute(
```

Comment:
@julienmouraddotcom, no need to have the protected variable here, we already have it in BaseSectionDisplayContext and it's available to you